src/multiscan_driver.cpp

```diff
@@ -5,6 +5,7 @@
 #include <vector>
 #include <deque>
 #include <limits>
```

Comment:
Try and remove this if it's not used (use RCLCPP_INFO logs instead of std::cout)